.circleci/config.yml

```diff
@@ -122,10 +105,10 @@ workflows:
   version: 2
   build-test-package:
     jobs:
-      - opencog
+      - pln
       - package:
           requires:
-            - opencog
+            - ure
```

Comment:
https://circleci.com/workflow-run/b78a0d57-e315-444c-9c0b-b0b9dfd51d58, reports 
> We weren't able to start this workflow.
package: requires job "ure" but "ure" is not part of this workflow.
The following jobs are unreachable: package

`ure` should be changed to `pln`